cypress/fixtures/fileLinks.js

```diff
@@ -0,0 +1,9 @@
+// we use real links for items
+// because we cannot return fixture urls?
+// todo: internal links? host our own links?
+
+export const MOCK_IMAGE_URL = 'https://picsum.photos/100';
+export const MOCK_VIDEO_URL =
+  'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4';
+export const MOCK_PDF_URL =
+  'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf';
```

Comment:
We should probaby spin up a very basic http file server (e.g. using apache or nginx) just to serve files locally for testing, for instance through a docker container